### ISSUE TITLE
paper-menu/paper-menu-button not working correctly

### ISSUE THREAD

**Issue by nishantgirbar** [hotforfeature/origami#64]
Hi,

paer-menu/paper-menu-button on Polymer 2.0 is not working as below code was working fine on anguler2/polymer.

<paper-menu-button vertical-offset="28" horizontal-align="right" class="selectedApp">
                <paper-icon-button icon="view-module" class="dropdown-trigger topbarIcon"></paper-icon-button>
                <paper-menu class="dropdown-content">
                    <paper-item *ngFor="let Db of GroupMenu">
                        <paper-button (click)="applicationRouting(Db.Name)">
                            <iron-icon icon="icons:all-out"></iron-icon> {{Db.Name}}</paper-button>
                    </paper-item>
                </paper-menu>
            </paper-menu-button>

basically paper-menu 2.0 is not available so is there any work around for that ?
"paper-menu": "PolymerElements/paper-menu#^2.0.0"

Thanks,
Nishant Kumar

**Comment by BorntraegerMarc** [comment]
@nishantgirbar we use `paper-menu-button` just fine in our project. Also I don't see any code in your issue. Did you forget to include it?

**Comment by nishantgirbar** [comment]
Thanks for response, please check below code, i don't know i was not able to paste code

**Comment by BorntraegerMarc** [comment]
Thx. Let me ask you couple of questions:
1. What do you mean with "it is not working". Is there an error message in the web console?
1. Did you follow the https://github.com/hotforfeature/origami/blob/master/README.md#installation docs properly to transition from angular 2 to 4 with polymer 2?

**Comment by hotforfeature** [comment]
[`<paper-menu>` was deprecated](https://github.com/PolymerElements/paper-menu/blob/master/README.md#paper-menu) in favor of `<paper-listbox>`. The element will not work in Polymer 2, though it would have worked in Polymer 1 (which is what the angular-polymer library supports).

You should be able to simply replace instances of `<paper-menu>` with `<paper-listbox>` for Polymer 2. Their apis are very similar. In addition, replace the `class="dropdown-content"` selector (Polymer 1) with `slot="dropdown-content"` (Polymer 2).

**Comment by nishantgirbar** [comment]
thanks a lot its working fine by replacing paper-menu with paper-listbox.

**Comment by hotforfeature** [comment]
Glad to hear that fixed it!

**Comment by BorntraegerMarc** [comment]
Oh that's right, good catch @hotforfeature !
